pkg/config/config_test.go

```diff
@@ -155,6 +157,51 @@ func TestNewConfig(t *testing.T) {
 	})
 }
 
+func Test_LoadConfig(t *testing.T) {
+	t.Run("valid-url", func(t *testing.T) {
+		c, err := loadConfig("https://raw.githubusercontent.com/get-woke/woke/main/example.yaml")
+		assert.NoError(t, err)
+		assert.NotNil(t, c)
+		// delete downloaded file after successful test
+		os.Remove("downloadedRules.yaml")
+	})
+
+	t.Run("invalid-url", func(t *testing.T) {})
+	_, err := loadConfig("https://raw.githubusercontent.com/get-woke/woke/main/example")
+	assert.Error(t, err)
+}
+
+func Test_isValidURL(t *testing.T) {
+	t.Run("valid-url", func(t *testing.T) {
+		boolResponse := isValidURL("https://raw.githubusercontent.com/get-woke/woke/main/example.yaml")
+		assert.True(t, boolResponse)
+	})
+
+	t.Run("invalid-url", func(t *testing.T) {
+		boolResponse := isValidURL("Users/Document/test.yaml")
+		assert.False(t, boolResponse)
+	})
+
+	t.Run("invalid-url", func(t *testing.T) {
+		boolResponse := isValidURL("/Users/Document/test.yaml")
```

Comment:
Out of curiosity, do you know if something like `C:\user\test.yaml` or `C:\directory.com\test.yaml` returns valid? Unsure how the URL validation works under the hood and just thinking of some weird edge cases

Reply:
it returns invalid  - adding in some more unit tests